ntp/syncTime.go

```diff
@@ -173,6 +185,15 @@ func (s *syncTime) sync() {
 				continue
 			}
 
+			if duration.Milliseconds() > maxAllowedNTPQueryResponseTimeMS {
+				log.Debug("sync.query exceeds maximum allowed response time",
```

Comment:
I think this is too much. for testing purposes it is ok, but this will flood the logs.

Reply:
also if the query take more time, can't we correct it ? maybe?

Reply:
1. Right, should be set to trace !
2. Not sure how to correct it, honestly. We do 10 requests/host anyway and take an average.

Reply:
like thinking about the difference between request and respons -> having it and taking that as corrected time ? :D